README.md

```diff
@@ -1 +1,283 @@
-# backend
+
+# hot-coffee-sql
+
+## Learning Objectives
+
+- SQL
+- PostgreSQL
+- CRUD
```

Comment:
Also, they've already learned the crud in foundation and even in sprint sql with basic crud and advanced ones at the end of the sprint and I think it would be better to make it Advanced CRUD and change requirements to have much more complicated endpoints, handling business logic and using complex queries that would require thorough planning of the tables as for example idea for one of the endpoints could the full text search:

**Full-Text Search**
   - Implement menu search using PostgreSQL's full-text search capabilities
   - Add endpoint `GET /menu/search` that allows searching by:
     * Item name
     * Ingredients
     * Description
     * Handling partial matches and typos